examples/linear_model/plot_sgd_comparison.py

```diff
@@ -21,22 +21,31 @@
 from sklearn.linear_model import LogisticRegression
 
 heldout = [0.95, 0.90, 0.75, 0.50, 0.01]
-rounds = 20
+rounds = 15
```

Comment:
I think we can change to 10 rounds and leave a comment describing what rounds do.

```suggestion
# Number of rounds to fit and evaluate an estimator.
rounds = 10
```